modules/op-starting-pipelines.adoc

```diff
@@ -19,7 +19,18 @@ For Pipelines created using the *From Git* option, the *Start Pipeline* dialog b
 ====
 +
 .. If you have resources in your namespace, the *Git Resources* and the *Image Resources* fields are prepopulated with those resources. If required, use the drop-downs to select or create the required resources and customize the Pipeline Run instance.
-.. If required, modify the *Advanced Options* and add credentials to authenticate the specified Git server or Docker registry.
+. Optional: Modify the *Advanced Options* and add credentials to authenticate the specified Git server or Docker registry.
++
+Under *Advanced Options*, click *Show Credentials Options* and select *Add Secret*. In the *Create Source Secret* section, specify the following:
+
+* A unique *Secret Name* for the secret.
+* In the *Designated provider to be authenticated* section, specify the provider to be authenticated in *Access to*, the base *Server URL*, and the *Authentication Type* to be used.
+* Specify the *Registry Server Address* that you want to authenticate, along with the credentials *Username*, *Password*, and *Email*.
```

Comment:
s/ to authenticate, along with/ to authenticate, and provide your credentials in the *Username*, *Password*, and *Email* fields.